src/autoplex/fitting/common/flows.py

```diff
@@ -300,6 +302,33 @@ def make(
                     logging.info(
                         f"File {file_name} has been copied to {destination_file_path}"
                     )
+            if len(self.pre_xyz_files) == 2:
+                # join to one file and then split again afterwards
+                # otherwise, split percentage will not be true
+                destination_file_path = os.path.join(
+                    current_working_directory, "vasp_ref.extxyz"
+                )
+                for file_name in self.pre_xyz_files:
+                    # TODO: if it makes sense to remove isolated atoms from other files as well
+                    atoms_list = ase.io.read(
+                        os.path.join(self.pre_database_dir, file_name), index=":"
+                    )
+                    new_atoms_list = [
+                        atoms
+                        for atoms in atoms_list
+                        if atoms.info["config_type"] != "IsolatedAtom"
+                    ]
+
+                    ase.io.write(destination_file_path, new_atoms_list, append=True)
+
+                    logging.info(
+                        f"File {self.pre_xyz_files[0]} has been copied to {destination_file_path}"
+                    )
```

Comment:
is this a good universal solution? maybe, i can reduce it to just one set of isolatedAtom energies to avoid failures